Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3-alpine
+
+WORKDIR /src
+COPY suisa_sendemeldung.py requirements.txt /src/
+RUN ["pip","install","-r","/src/requirements.txt"]
```

Comment:
I chose the exec over the shell form because I thought it would not be necessary to run an additional `sh` process in the build process. I'm not sure what the best practice is on this though, since the [documentation](https://docs.docker.com/engine/reference/builder/#run) does not say a thing.

`--no-cache-dir` does make sense of course.

Reply:
Makes sense, fie with me.